src/transactions/feed/TransactionFeed.tsx

```diff
@@ -36,16 +36,37 @@ function TransactionFeed() {
   const allowedNetworks = getAllowedNetworkIds()
 
   const confirmedFeedTransactions = useMemo(() => {
+    // Filter out received pending transactions that are also in the pending
+    // standby array because those will be displayed with the pending
+    // transactions
+    const completedTransactionsOrWithoutPendingStandby = transactions.filter(
```

Comment:
nit: name is a little confusing, maybe `completedOrNotPendingStandbyTransactions`? still kind of a mouthful either way :sweat_smile:

Reply:
haha yeah i like it, thanks!